tests/Unit/SDK/Resource/Detectors/HostTest.php

```diff
@@ -21,5 +22,60 @@ public function test_host_get_resource(): void
         $this->assertSame(ResourceAttributes::SCHEMA_URL, $resource->getSchemaUrl());
         $this->assertIsString($resource->getAttributes()->get(ResourceAttributes::HOST_NAME));
         $this->assertIsString($resource->getAttributes()->get(ResourceAttributes::HOST_ARCH));
+        $this->assertIsString($resource->getAttributes()->get(ResourceAttributes::HOST_ID));
+    }
+
+    public function test_host_parse_macos_id(): void
+    {
+        $out = 'IOPlatformUUID=1234567890';
```

Comment:
since the parsing function allows for multiple lines, perhaps this could be a more realistic multi-line input?